src/libsiedler2.cpp

```diff
@@ -238,7 +238,8 @@ int LoadFolder(std::vector<FileEntry> folderInfos, Archiv& items, const ArchivIt
             {
                 if(static_cast<unsigned>(entry.nr) < items.size() && items[entry.nr]->getBobType() == BobType::Palette)
                     curPal = static_cast<const ArchivItem_Palette*>(items[entry.nr]);
-            } else if(!items.empty() && items[items.size() - 1u]->getBobType() == BobType::Palette)
+            } else if(!items.empty() && items[items.size() - 1u]
```

Comment:
istn items.back() the same?

Reply:
`items` is an archive and that doesn't have `back`

Reply:
ah okay. I thought its an `vector`